public/stylesheets/navbar.css

```diff
@@ -0,0 +1,108 @@
+/* General Navbar Styles */
+.navbar {
+    background-color: black; /* Black background for the navbar */
+    transition: background-color 0.3s ease-in-out;
+    height: 60px; /* Set consistent height */
+    width: 100%; /* Ensure navbar takes full width */
+}
+
+.navbar-brand {
+    font-weight: bold; /* Bold CampX logo */
+    color: white; /* White logo text */
+    font-size: 2rem;
+}
+
+.navbar-brand:hover {
+    color: white; /* Ensure the CampX logo stays white on hover */
+}
+
+.navbar-brand #rotateX {
+    display: inline-block;
+    animation: rotation 3s infinite;
+    color: white; /* Color of rotating X */
+}
+
+.navbar-nav .nav-link {
+    color: rgba(255, 255, 255, 0.5); /* Semi-transparent text */
+    font-weight: 700;
+    margin-left: 1rem;
+    transition: color 0.3s ease;
+}
+
+.navbar-nav .nav-link:hover {
+    color: rgba(255, 255, 255, 0.7); /* Slightly brighter text on hover */
+}
+
+.navbar-nav .nav-link.active {
+    color: white !important; /* Active link stays white */
```

Comment:
even without !important , its still white. Then why do you have !important?

Reply:
Removed now